tests/system/feature_store/test_feature_validator.py

```diff
@@ -9,76 +13,209 @@
 @TestMLRunSystem.skip_test_if_env_not_configured
 class TestFeatureValidator(TestMLRunSystem):
     def validator_types(self):
-        for validator in validators_check_type.keys():
-            for test_state, test_value in validators_check_type[validator]:
+        for validator in self._validators_check_type.keys():
+            for test_state, test_value in self._validators_check_type[validator]:
                 feature = Feature(validator)
                 feature.validator = Validator(True, "Info")
 
                 feature.validator.set_feature(feature)
                 ok, result = feature.validator.check(test_value)
                 assert ok == test_state, f"Check type error, result:{result}"
 
+    @TestMLRunSystem.skip_test_if_env_not_configured
+    @pytest.mark.enterprise
+    def feature_set_entities(self):
```

Comment:
I tested only the logic in functions, but I am not able to run these tests directly in MLRun project because:

- IT security restriction/policy for installation WSL2 (it needs allow windows upgrade)
- I cannot access to the environment in country, because access via firewalls (and the Iguazio platform was installed in different country)

I would like to get relevant roles for installation of WSL2 for desktop docker but it will takes some times.

Reply:
Done